packages/framework/client-logger/app-insights-logger/package.json

```diff
@@ -29,13 +29,11 @@
 	"scripts": {
 		"api": "fluid-build . --task api",
 		"api-extractor:commonjs": "api-extractor run --local",
-		"api-extractor:esnext": "copyfiles -u 1 \"dist/**/*-@(alpha|beta|public|untrimmed).d.ts\" lib",
```

Comment:
This package doesn't need the ESM version of this? It does have an ESM build.

Reply:
This was a mistake. Fixed in latest.